src/antsibull_nox/ee.py

```diff
@@ -0,0 +1,161 @@
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or
+# https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2025, Ansible Project
+
+"""
+Build execution environments for testing.
+"""
+
+from __future__ import annotations
+
+import argparse
+import dataclasses
+import os
+import subprocess
+from pathlib import Path
+from typing import Optional
+
+import yaml
```

Comment:
Instead of importing `yaml`, better use `from antsibull_fileutils.yaml import load_yaml_file, store_yaml_file` and call these functions.